### ISSUE TITLE
Release 2.1.1 - ButterKnife changes

### ISSUE THREAD

**Issue by jcdom** [Karumi/Rosie#86]
When you upgrade version 2.1.0 to 2.1.1, you have to add code to keep ButterKnife running, because ButterKnife and Renderer libraries has been moved to the sample application in this commit: https://github.com/Karumi/Rosie/commit/0ab7fc39fc161878ac377d211d10860174dc988e

First, you have to add ButterKnife dependency:
`compile 'com.jakewharton:butterknife:7.0.1'`

Second, you have to add `ButterKnife.bind(this);` to your activities or base activity (and fragments).
Rosie provides a new method to do it:
```java
@Override
protected void onPrepareActivity() {
    super.onPrepareActivity();
    ButterKnife.bind(this);
}
```

@flipper83 Maybe, this should be added to the release changes

**Comment by flipper83** [comment]
Hi, 

Sorry, we are going to fix it with two steps, we are going to launch a new revision version without breaking changes and a new mayor version with te breaking changes and document the breaking change in the release changes.

Sorry for the inconvenience.

**Comment by Serchinastico** [comment]
Hi @jcdom 

As we promised, we just released v2.1.2 reverting the removal of the libraries and v3.0.0 with re-reverting that same change ( 😅 ). In your case, if you have already done the change I recommend you to use v3.0.0.

We are sorry for the inconveniences.
Thank you for your time, reporting and patience
